src/cli.js

```diff
@@ -39,6 +39,11 @@ yargs
 			type: 'string',
 			defaultDescription: './build',
 			description: 'Directory for build artifacts'
+		},
+		publicpath: {
+			type: 'string',
+			defaultDescription: '/',
+			description: 'Public path you would like to download resources from, e.g. "/@zimbra/service/zimlet/com_example_zimlet/"'
```

Comment:
not sure why we need this as configuration
this path depends on zimlet name, so you can't set it different then zimlet name

Reply:
Peter said that the user might want to use a CDN link too. Also this is at build time. We specify zimlet name at the time of packaging so I thought i'd allow path here.
@silentsakky

Reply:
Yea the idea is that if we are no longer supporting the `/service/zimlet/` routes, then we will only be able to use CDN so we will have to have the developer provide their `publicPath` at build time.